hledger-web/hledger-web.cabal

```diff
@@ -103,6 +103,7 @@ extra-source-files:
     templates/edit-form.hamlet
     templates/journal.hamlet
     templates/manage.hamlet
+    templates/piechart.hamlet
```

Comment:
Would you mind making this same change in the package.yaml file, from which this is generated. (Changing the .cabal file as well is optional, stack does it and I sometimes commit that change separately to reduce conflicts.)